src/imitation/scripts/common/reward.py

```diff
@@ -20,12 +21,37 @@ def config():
     locals()  # quieten flake8
 
 
+@reward_ingredient.named_config
+def normalize_disable():
```

Comment:
Assuming we'll want to add output normalization too at some point, we might want to use a different name for these named configs right away. E.g. `normalize_obs_disable` and `normalize_obs_running`, then we can later have similar options for `normalize_rew`. Could also rename `normalize_layer` to e.g. `normalize_input_layer` (since we'll later need `normalize_output_layer`).

OTOH, if we don't end up adding reward normalization for some reason, keeping the current names seems fine. So if you're very uncertain about that, could also change this in another PR

Reply:
Good suggestion, changed.